go/pkg/bertymessenger/api.go

```diff
@@ -85,10 +87,9 @@ func (svc *service) internalInstanceShareableBertyID(ctx context.Context, req *I
 		return nil, errcode.TODO.Wrap(err)
 	}
 
-	displayName := req.DisplayName
+	displayName := strings.TrimSpace(req.DisplayName)
 	if displayName == "" {
-		// FIXME: get it from somewhere
-		displayName = "anonymous#1337"
+		return nil, errcode.TODO
```

Comment:
maybe don't return an error since a client might want to not use a name and the links format supports it